src/System.Threading.Thread/tests/ThreadTests.cs

```diff
@@ -1025,5 +1025,49 @@ public static void MiscellaneousTest()
             Thread.SpinWait(1);
             Thread.Yield();
         }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public static void WindowsPrincipalPolicyTest_Windows()
+        {
+            DummyClass.RemoteInvoke(() =>
+            {
+                if (!string.IsNullOrEmpty(Environment.UserDomainName))
```

Comment:
What does it mean if the UserDomainName is null or empty?  When can that happen?

Reply:
Intended for case where Machine is not joined to the domain or user is not domain user (but perhaps that returns the machine name?)

Reply:
OK we don't need the null/empty check then.